MigrationProject/MigrationProjectTests/MigrationProjectTests.m

```diff
@@ -0,0 +1,71 @@
+//
+//  MigrationProjectTests.m
+//  MigrationProjectTests
+//
+//  Created by Brock Boland on 10/26/16.
+//  Copyright © 2016 Vokal. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+
+#import <VOKCoreDataManager.h>
+
+@interface MigrationProjectTests : XCTestCase
+
+@property (nonatomic) NSURL *persistentStoreURL;
+
+@end
+
+@implementation MigrationProjectTests
+
+- (void)setUp
+{
+    [super setUp];
+
+    [[VOKCoreDataManager sharedInstance] resetCoreData];
+
+    // Load a copy of the DB that used the first version of the model
+    NSBundle *bundle = [NSBundle bundleForClass:[self class]];
+    NSURL *oldPersistentStoreURL = [bundle URLForResource:@"MyDB" withExtension:@"sqlite"];
+
+    XCTAssertNotNil(oldPersistentStoreURL, @"Cannot find MyDB.sqlite");
+
+    NSURL *docsDir = [[NSFileManager defaultManager] URLsForDirectory:NSDocumentDirectory inDomains:NSUserDomainMask].lastObject;
+    self.persistentStoreURL = [docsDir URLByAppendingPathComponent:@"testdb"];
+    
+    // Copy the old version of the database into the documents directory to work with
+    NSError *error;
+     [[NSFileManager defaultManager] copyItemAtURL:oldPersistentStoreURL
+                                             toURL:self.persistentStoreURL
+                                             error:&error];
+     XCTAssertNil(error);
```

Comment:
in b4 @vokal-isaac talks about how you should also check the result of this and not just the error - a nil error is not a guarantee of success